.github/workflows/ci.yml

```diff
@@ -58,6 +58,8 @@ jobs:
       - name: Build
         run: |
           PKG_CONFIG_PATH=/tmp/fcitx5/lib/pkgconfig cmake -B build -G Ninja \
+            -DWKWEBVIEW_PROTOCOL="fcitx" \
```

Comment:
actually I think we can simplify this bit. Because fcitx-contrib/webview is our own fork, and is expected to be used only by us.

Reply:
This serves 2 purposes:
* use WEBVIEW2_PROTOCOL and GTKWEBVIEW_PROTOCOL to control platform-specific code in the future.
* Keep the repo generic by making it optional.

Reply:
but it won't be optional after merging https://github.com/fcitx-contrib/fcitx5-macos/pull/166 ?

> GTKWEBVIEW_PROTOCOL
does this mean we have to use something other than `fcitx://` on Linux?

Reply:
> but it won't be optional after merging

Mandatory for f5m, optional for potentially other IMEs. If someone wants to fork Squirrel and replace its candidate window, I'd be happy to see that. After all we have the best scroll mode in Open Source IMEs.

> we have to use something other than fcitx://

No, but the idea is to use `GTKWEBVIEW_PROTOCOL="fcitx"` on Linux, so that our fork of webview will have code like:
```c++
cocoa_funcion() {
#ifdef WKWEBVIEW_PROTOCOL
...
#endif
}

gtk_function() {
#ifdef GTKWEBVIEW_PROTOCOL
...
#endif
}
```

Reply:
good point